docs/targets/googlesheets.md

```diff
@@ -1,21 +1,15 @@
-# Google spreadsheets Target
+# Event target for Google Sheets
 
-## Using the sink adapter
+This event target receives [CloudEvents][ce] over HTTP and sends them to GoogleSheets.
```

Comment:
Same comment as in another PR: we should be slightly more accurate. We don't "send events" to GS, we "use the event payload to populate a Sheet".